mlir/utils/jenkins/Jenkinsfile

```diff
@@ -185,7 +185,7 @@ String getLabelFromCodepath(String codepath) {
     } else if (codepath == "vanilla"){
         label = 'mlir'
     } else if (codepath == "navi3x") {
-        label = 'mlir && ( gfx1100 || gfx1101 )'
```

Comment:
First controversial part.  In my recent experience, gfx1101 has been less likely to have problems than gfx1100.  Let's experiment with using it.